### ISSUE TITLE
Does not install on Node 8.x LTS

### ISSUE THREAD

**Issue by pebcakerror** [majodev/google-webfonts-helper#48]
Node version 8.10.0

```
> npm install

> v8-profiler@5.2.9 preinstall /Users/fredderf/project/web-design-contest-fonts/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/v8-profiler
>


> v8-debug@0.4.6 preinstall /Users/fredderf/project/web-design-contest-fonts/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/v8-debug
>


> bson@0.2.22 install /Users/fredderf/project/web-design-contest-fonts/node_modules/connect-mongo/node_modules/mongodb/node_modules/bson
> (node-gyp rebuild 2> builderror.log) || (exit 0)

  CXX(target) Release/obj.target/bson/ext/bson.o

> kerberos@0.0.11 install /Users/fredderf/project/web-design-contest-fonts/node_modules/connect-mongo/node_modules/mongodb/node_modules/kerberos
> (node-gyp rebuild 2> builderror.log) || (exit 0)

  CXX(target) Release/obj.target/kerberos/lib/kerberos.o

> v8-debug@0.4.6 install /Users/fredderf/project/web-design-contest-fonts/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/v8-debug
> node-pre-gyp install --fallback-to-build

node-pre-gyp info it worked if it ends with ok
node-pre-gyp verb cli [ '/Users/fredderf/.nvm/versions/node/v8.9.1/bin/node',
node-pre-gyp verb cli   '/Users/fredderf/project/web-design-contest-fonts/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/v8-debug/node_modules/.bin/node-pre-gyp',
node-pre-gyp verb cli   'install',
node-pre-gyp verb cli   '--fallback-to-build' ]
node-pre-gyp info using node-pre-gyp@0.6.9
node-pre-gyp info using node@8.9.1 | darwin | x64
node-pre-gyp verb command install []
node-pre-gyp info check checked for "/Users/fredderf/project/web-design-contest-fonts/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/v8-debug/build/debug/v0.4.6/node-v57-darwin-x64/debug.node" (not found)
node-pre-gyp http GET https://node-inspector.s3.amazonaws.com/debug/v0.4.6/node-v57-darwin-x64.tar.gz
node-pre-gyp ERR! UNCAUGHT EXCEPTION
node-pre-gyp ERR! stack Error: Cannot find module 'internal/util/types'
node-pre-gyp ERR! stack     at Function.Module._resolveFilename (module.js:536:15)
node-pre-gyp ERR! stack     at Function.Module._load (module.js:466:25)
node-pre-gyp ERR! stack     at Module.require (module.js:579:17)
node-pre-gyp ERR! stack     at require (internal/module.js:11:18)
node-pre-gyp ERR! stack     at evalmachine.<anonymous>:31:26
node-pre-gyp ERR! stack     at Object.<anonymous> (/Users/fredderf/project/web-design-contest-fonts/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/v8-debug/node_modules/node-pre-gyp/node_modules/tar/node_modules/fstream/node_modules/graceful-fs/fs.js:11:1)
node-pre-gyp ERR! stack     at Module._compile (module.js:635:30)
node-pre-gyp ERR! stack     at Object.Module._extensions..js (module.js:646:10)
node-pre-gyp ERR! stack     at Module.load (module.js:554:32)
node-pre-gyp ERR! stack     at tryModuleLoad (module.js:497:12)
node-pre-gyp ERR! System Darwin 17.4.0
node-pre-gyp ERR! command "/Users/fredderf/.nvm/versions/node/v8.9.1/bin/node" "/Users/fredderf/project/web-design-contest-fonts/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/v8-debug/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /Users/fredderf/project/web-design-contest-fonts/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/v8-debug
node-pre-gyp ERR! node -v v8.9.1
node-pre-gyp ERR! node-pre-gyp -v v0.6.9
node-pre-gyp ERR! This is a bug in `node-pre-gyp`.
node-pre-gyp ERR! Try to update node-pre-gyp and file an issue if it does not help:
node-pre-gyp ERR!     <https://github.com/mapbox/node-pre-gyp/issues>
npm ERR! code ELIFECYCLE
npm ERR! errno 7
npm ERR! v8-debug@0.4.6 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 7
npm ERR!
npm ERR! Failed at the v8-debug@0.4.6 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/fredderf/.npm/_logs/2018-03-27T00_41_10_772Z-debug.log
```

**Comment by majodev** [comment]
see `.nvmrc` --> `0.10.44`, I know this is quite an old project now. Supporting [Node.js LTS](https://github.com/nodejs/Release) is not planned currently, maybe with the next full refactor.

**Comment by DanielRuf** [comment]
> see .nvmrc --> 0.10.44, I know this is quite an old project now. Supporting Node.js LTS is not planned currently, maybe with the next full refactor.

I could help with this. Also fixing builds should be possible =)

**Comment by DanielRuf** [comment]
`grunt-node-inspector/node_modules/node-inspector/node_modules/v8-debug`

Probably just an outdated dependency.

Try updating Node.js and npm to the latest releases and clear the npm cache.
https://stackoverflow.com/questions/40308623/cannot-find-module-internal-fs-after-upgrading-to-node-7/40406482#40406482

**Comment by backflip** [comment]
I had to remove the `grunt-node-inspector` and used Node 6.

**Comment by majodev** [comment]
Fixed within [v1.1.0](https://github.com/majodev/google-webfonts-helper/releases/tag/v1.1.0), uses node@v18.
